### ISSUE TITLE
Option to reverse volume direction

### ISSUE THREAD

**Issue by edyman01** [t3knomanzer/maxmix-software#192]
**Guidelines**
- [x] I have reviewed the [guidelines for contributing](https://github.com/t3knomanzer/maxmix-software/blob/master/.github/CONTRIBUTING.md) to this repository.

**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
I would like the volume to increase when I turn clockwise and decrease when I turn counterclockwise. An option to change this in the application would be useful.

**Describe alternatives you've considered**

**Additional context**

**Comment by ItsLemmy** [comment]
That is very surprising, it's supposed to increase when turning clockwise....
Do you have a link to the knob/part you bought ?

**Comment by edyman01** [comment]
RON 2.71 5%OFF | Rotary Encoder Module for Free Shipping Dropshipping KY-040
https://a.aliexpress.com/_mr4Xa3V

**Comment by pbanj** [comment]
you sure the wires are plugged up correctly?

**Comment by t3knomanzer** [comment]
As @pbanj mentioned. Could you check that your Dt and Clk are connected to the right pins?   
Clockwise to increase volume and counter-clockwise to reduce volume is how it’s meant to work.

**Comment by edyman01** [comment]
Yes, that was the first thing I checked.  The wires are connected correctly.

**Comment by t3knomanzer** [comment]
That's just inexplicable. If the wires are connected correctly and you're using the correct encoder, it should raise clockwise and decrease conunter-clockwise.

I wonder if the missing R1 has something to do with this.

![image](https://user-images.githubusercontent.com/2942293/93047981-a1a4d580-f612-11ea-8136-3cf31c77863c.png)

Can you confirm that if your encoder looks exactly like the one in the photo?

Sorry for asking so many questions but this is a behavior we've never seen before and it's not caused by the software so it is definitely something on your hardware. With the wiring being discarded the only other option is the encoder itself.

**Comment by edyman01** [comment]
if the picture is on the back of the encoder I can't check, I glued the
encoder to the bottom plate (both holes for screws didn't fit). :))

În lun., 14 sept. 2020 la 08:46, Ruben Henares <notifications@github.com> a
scris:

**Comment by SchemingWeasels** [comment]
I think adding support for inverting rotation direction should be quite simple

**Comment by edyman01** [comment]
If memory allows this option would be perfect!

În mar., 15 sept. 2020 la 19:49, SchemingWeasels <notifications@github.com>
a scris:

**Comment by t3knomanzer** [comment]
It should. We have cleaned up quite a bit of memory. 

Ruben Henares

**Comment by GastonAlgaze** [comment]
Same thing happened to me...
